cypress/integration/smoke.test.ts

```diff
@@ -5,41 +5,34 @@ if (!EMAIL || !PASSWORD) {
   throw new Error('You must provide CYPRESS_USER_EMAIL and CYPRESS_USER_PASSWORD environment variables');
 }
 
-const loginToAuth0 = (): void => {
-  cy.visit('/');
-  cy.get('#login').click();
-  cy.get('.auth0-lock-input-email input').focus().clear().type(EMAIL);
-  cy.get('.auth0-lock-input-password input').focus().clear().type(PASSWORD);
-  cy.get('.auth0-lock-submit').click();
-};
+describe('smoke tests', () => {
+  before(() => {
+    cy.visit('/');
+    cy.get('[data-testid=login]').click();
+    cy.get('input[name=email], input[name=username]').focus().clear().type(EMAIL);
+    cy.get('input[name=password]').focus().clear().type(PASSWORD);
+    cy.get('button[name=submit], button[name=action]').click();
+  });
 
-describe('Smoke tests', () => {
-  it('should do basic login and show user', () => {
-    loginToAuth0();
+  after(() => {
+    cy.get('[data-testid=logout]').click();
+  });
 
+  it('should do basic login and show user', () => {
     cy.url().should('eq', `${Cypress.config().baseUrl}/`);
-    cy.get('#profile').contains(EMAIL);
-    cy.get('#logout').click();
-    cy.get('#login').should('exist');
```

Comment:
Can we include a test for a successful logout?

Reply:
Done, thanks for pointing this out.